src/os/os_unix_test.go

```diff
@@ -0,0 +1,57 @@
+// +build darwin linux,!baremetal freebsd,!baremetal
+
+package os_test
+
+import (
+	"os"
+	. "os"
+	"testing"
+)
+
+func TestMkdir(t *testing.T) {
+	// FIXME: put test files in unique directory
+	foo := "/tmp/_Test_Mkdir_.tmp"
```

Comment:
I'm worried that this will cause trouble.
I think we should at the very least try to make a unique directory based on the current time, possibly including the PID. Alternatively, crypto/rand could perhaps be used for a unique name.

Reply:
I'll implement MkdirTemp and use that, I think.  
First step: https://github.com/tinygo-org/tinygo/pull/2266

Reply:
No crypto/rand for me, alas:
```
	imports os
	imports crypto/rand
	imports math/big
	imports fmt
	imports os: import cycle not allowed
```
On to door #2.

Reply:
Huh, weird. This shouldn't happen: the test package is named os_test.

Reply:
Just using time and pid for now.